hphp/system/php/redis/Redis.php

```diff
@@ -1650,7 +1650,7 @@ protected function doConnect($host,
         E_WARNING);
       return false;
     }
-    stream_set_blocking($this->connection, true);
+    stream_set_blocking($this->connection, 1);
```

Comment:
http://php.net/manual/en/function.stream-set-blocking.php <-- docs say it should be bool